docs/monitoring.md

```diff
@@ -119,6 +119,14 @@ The history server can be configured as follows:
     </td>
   </tr>
   <tr>
+    <td>spark.history.ui.maxApplications</td>
+    <td>Int.MaxValue</td>
+    <td>
+      The number of applications to display on the history summary page. Application UIs are
+      still available even if they are not displayed on the history summary page.
```

Comment:
we might want to clarify this, I assume its if you put url to the application directly